spec/requests/articles/articles_update_spec.rb

```diff
@@ -94,7 +94,6 @@
 
   it "removes all published notifications if unpublished" do
     user2.follow(user)
-    Notification.send_to_followers_without_delay(article, "Published")
```

Comment:
Maybe the intentions of this line are not clear but it's important for the test. Actually, it creates the notifications, and later we check that they are destroyed. I suggest creating notifications manually for this test instead of the old line.